src/libraries/Common/Common.Tests.sln

```diff
@@ -11,10 +11,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{C72FD34C-539A-4447-9796-62A229571199}.Debug|Any CPU.ActiveCfg = netcoreapp5.0-Windows_NT-Debug|Any CPU
```

Comment:
Why?

Reply:
VS no longer knows that this is a configuration and dont know how to parse it which was crashing the VS.
Now the Configuration can only be Debug or Release